### PR TITLE
fix providing wrong evm version

### PR DESCRIPTION
**Pull request by yann300**
fix https://github.com/ethereum/remix-project/issues/1970

### CODE REVIEW

File: libs/remix-ui/solidity-compiler/src/lib/logic/compileTabLogic.ts [hunk 1]
```diff
@@ -19,10 +19,12 @@ export class CompileTabLogic {
   public language: string
   public compilerImport
   public event
+  public evmVersions: Array<string>
 
   constructor (public api: ICompilerApi, public contentImport) {
     this.event = new EventEmitter()
     this.compiler = new Compiler((url, cb) => api.resolveContentAndSave(url).then((result) => cb(null, result)).catch((error) => cb(error.message)))
+    this.evmVersions = ['default', 'london', 'istanbul', 'petersburg', 'constantinople', 'byzantium', 'spuriousDragon', 'tangerineWhistle', 'homestead']
```
Comment (Aniket-Engg) [inline libs/remix-ui/solidity-compiler/src/lib/logic/compileTabLogic.ts:27]: `berlin` is missing in the list

Comment (yann300) [reply libs/remix-ui/solidity-compiler/src/lib/logic/compileTabLogic.ts]: that's fixed

**Review by Aniket-Engg — COMMENTED** [review]
With IPFS env, it just doesn't set evm version from url params but just set `null` in url and `default` in the list

**Review by Aniket-Engg — APPROVED** [review]
Looks fine to me!

**Pull request merged**